corehq/apps/style/static/style/js/hq_extensions.jquery.js

```diff
@@ -0,0 +1,52 @@
+(function () {
```

Comment:
This is a direct move - the only thing I changed was putting the comments next to the appropriate copypastas.